homeassistant/components/seventeentrack/services.yaml

```diff
@@ -0,0 +1,11 @@
+get_packages:
+  fields:
+    package_state:
+      example: "In Transit,Delivered"
```

Comment:
I think we can make this a select selector since we already know all possible values

Reply:
Done